open-lens/src/main/index.ts

```diff
@@ -36,6 +37,7 @@ runInAction(() => {
     messagingFeatureForMain,
     randomFeature,
     kubeApiSpecificsFeature,
+    notificationsFeature,
```

Comment:
I don't think that notifications exist on `main`, so we probably don't need this.